eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/utils/charsequence/LeadingEmptyLinesAndTrailTrimming.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.utils.charsequence;
+
+import com.schedjoules.eventdiscovery.framework.utils.factory.Factory;
+
+
+/**
+ * {@link CharSequence} decorator that trims leading empty lines and trims the end of the text.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class LeadingEmptyLinesAndTrailTrimming extends AbstractCharSequence
```

Comment:
Ok, I will use the `Trimmed` `CharSequence` then if you think that's better. And remove this one.